apps/dapp/src/components/Pages/Core/DappPages/SpiceBazaar/Analytics/Chart/ValueOfSpiceAuc.tsx

```diff
@@ -1,95 +1,62 @@
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
 import styled, { useTheme } from 'styled-components';
 import { subDays } from 'date-fns';
 import { format } from 'date-fns';
 import { formatNumberAbbreviated } from 'utils/formatter';
-import { InputSelect } from 'components/InputSelect/InputSelect';
-import CustomBarChart from './BarChart';
+import CustomBarChart from '../components/BarChart';
+import * as breakpoints from 'styles/breakpoints';
+import { InputSelect, Option } from '../../components/InputSelector';
+import Loader from 'components/Loader/Loader';
 
 type XAxisTickFormatter = (timestamp: number) => string;
 
 export const tickFormatter: XAxisTickFormatter = (timestamp) =>
   format(new Date(timestamp), 'MMM dd');
 
-export type Metric = {
-  timestamp: number;
+type Metric = {
+  name: string;
   value1: number;
   value2: number;
   value3: number;
   value4: number;
 };
 
-const pricesLast7Days: Metric[] = [
+const data = [
```

Comment:
could we pull this static data from an API or other source?

Reply:
Yes this is on the analytics page, which is incomplete but still included in this PR. I started building it but paused to focus on MVP.

So, this is not visible and when we ship it, it will be pulled from the subgraph or other sources.